auth-web/src/components/auth/staff/account-management/StaffActiveAccountsTable.vue

```diff
@@ -1,63 +1,180 @@
+
+                      <div v-else-if="['orgType'].includes(header.value)" class="mt-0">
+                          <v-select
+                            :items="accountTypes"
+                            v-model="searchParams[header.value]"
+                            filled
+                            item-text="description"
+                            item-value="code"
+                            return-object
+                            data-test="select-status"
+                            v-bind="$attrs"
+                            v-on="$listeners"
+                            hide-details="auto"
+                            :placeholder="header.text"
+                          />
+                      </div>
+
+                      <v-btn v-else-if="searchParamsExist && header.value === 'action'"
+                        outlined
+                        color="primary"
+                        class="action-btn"
+                        @click="clearSearchParams()"
+                      >
+                        Clear Filters
+                      </v-btn>
+                    </th>
+                  </tr>
+                </thead>
+              </template>
+
+              <template v-slot:[`item.orgType`]="{ item }">
+                  {{getAccountTypeFromOrgAndAccessType(item)}}
+              </template>
+              <template v-slot:[`item.decisionMadeBy`]="{ item }">
+                  {{item.decisionMadeBy ? item.decisionMadeBy : 'N/A'}}
+              </template>
+
+               <!-- Item Actions -->
+              <template v-slot:[`item.action`]="{ item }">
+                <div class="actions text-right">
+                  <span class="open-action">
+                    <v-btn
+                      color="primary"
+                      class="open-action-btn"
+                      @click="view(item)"
+                      :data-test="getIndexedTag('view-account-button', item.id)"
+                    >
+                      View
+                    </v-btn>
+                  </span>
+
+                  <!-- More Actions Menu -->
+                  <span class="more-actions">
+                    <v-menu
+                      v-model="dropdown[item.id]"
+                    >
+                      <template v-slot:activator="{ on }">
+                        <v-btn
+                          color="primary"
+                          class="more-actions-btn"
+                          v-on="on"
+                        >
+                          <v-icon>{{dropdown[item.id] ? 'mdi-menu-up' : 'mdi-menu-down'}}</v-icon>
+                        </v-btn>
+                      </template>
+                      <v-list>
+                        <v-list-item @click="viewInBusinessRegistryDashboard(item)">
+                          <v-list-item-subtitle>
+                            <v-img class="business-dashboard-icon" src="@/assets/img/StaffActive_business_dashboard_icon.svg"/>
```

Comment:
use mdi icons. This icon is available in mdi icon

Reply:
Ahh ok, I looked and couldn't find it.. any idea the name?

Reply:
I think it will be mdi-view-dashboard https://pictogrammers.github.io/@mdi/font/6.5.95/